### ISSUE TITLE
Getting 404

### ISSUE THREAD

**Issue by jsifalda** [stpe/instagram-slack#2]
Hello I run your script on my heroku app but keep getting this:
https://i.imgur.com/WaonxC8.jpg
https://i.imgur.com/n4Pcm3k.jpg
any ideas?

**Comment by stpe** [comment]
The screenshot indicates that Instagram is returning a 404 Page not found
error. Have you set the required environment variables on Heroku?

**Comment by jsifalda** [comment]
My profile is instagram.com/jsifalda, so I put there "jsifalda"... is it correct?

**Comment by stpe** [comment]
No, that is your username. You must use your Instagram user ID. (simple way
to find out; view source on your profile web page and find the "owner"
property in the embedded JSON).

**Comment by jsifalda** [comment]
Ok, i found my user ID. Thanks for the help. However, now it says "The access_token provided is invalid." https://i.imgur.com/AI3k2ep.jpg and I am sure I generated my access token correctly. (based on this tutorial https://stackoverflow.com/a/28559830)... maybe because my app is in sandbox mode?
